### PR TITLE
Service Monitoring

### PR DESCRIPTION
**Pull request by farismalaeb**
# PR Summary

## PR Checklist

- Do you want this post to be published on a specific date?: Y/N [Specify date]:N
- [X] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [X] PR has a meaningful title
- [X] Includes content related an open issue
  - see [Closing issues using keywords][key].
- [X] This PR is ready to merge and is not **Work in Progress**

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+- Create the WMI query
+- Registering the query
+- Reading the events.
+
+## Creating WMI Query
+
+WMI database has several system WMI Classes.
+For instance, the *CIM_InstModification* monitors the targeted class for any changes.
+So, in this case, *Win32_Service*.
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:83]: For example, you can use the **CIM_InstModification** class to monitor the targeted class, in this case **Win32_Service**

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+The query syntax looks like
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:85]: You have to create a WMI query using WMI Query Langaage. 

Can you add a reference to WQL?

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+```powershell-console
+Select * from <WMI System Class> within <Number of Seconds> where TargetInstance ISA <WMI Class name>
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:88]: SELECT * FROM <WMI System Class> WITHIN <Number of Seconds> WHERE TargetInstance ISA <WMI Class name>

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+```
+
+Let apply the same to *Win32_Serivce.* Start by Creating a PowerShell variable name it $Query and type the following query
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:91]: In our case, you construct the query as follows:

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+
+```powershell-console
+PS C:\> $query = "Select * from CIM_InstModification within 10 where TargetInstance ISA 'Win32_Service'"
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:94]: PS> $Query = "SELECT * from CIM_InstModification WITHIN 10 WHERE TargetInstance ISA 'Win32_Service'"

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+```
+
+> A full explanation for the WQL query is available in [Your Goto Guide for Working with Windows WMI Events and PowerShell](https://adamtheautomator.com/your-goto-guide-for-working-with-windows-wmi-events-and-powershell/)
+
+## Registering The Query
+
+We have the WQL query, let move to the next step and register the query to the WMI events by using the **Register-CimIndicationEvent**
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:101]: Explain what this command does

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+
+```powershell-console
+PS C:\> Register-CimIndicationEvent -Namespace 'ROOT\CIMv2' -Query $query -SourceIdentifier 'WindowsServices' -MessageData 'Service Status Change'
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:104]: PS > Register-CimIndicationEvent -Namespace 'ROOT\CIMv2' -Query $Query -SourceIdentifier 'WindowsServices' -MessageData 'Service Status Change'

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 8]
```diff
@@ -0,0 +1,209 @@
...
+```
+
+To confirm the successful registration, type the following cmdlet *Get-EventSubscriber,* the output looks like the following
+
+```powershell-console
+PS C:\> Get-EventSubscriber
+```
+
+```powershell
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:113]: remove these 4 lines and just have one console block

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 9]
```diff
@@ -0,0 +1,209 @@
...
+PS C:\> Get-EventSubscriber
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:114]: PS> Get-EventSubscriber

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 10]
```diff
@@ -0,0 +1,209 @@
...
+Now the event is registered and active.
+Next, create an event, and by that, I mean stopping or starting a service.
+
+Try *Windows Update* service (wuauserv), run the following cmdlet to see the status of the bits service.
+
+```powershell-console
+PS C:\> Get-Service wuauserv
+```
+
+```powershell
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:139]: remove these 4 lines

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 11]
```diff
@@ -0,0 +1,209 @@
...
+
+So the service is running, let stop it by typing the following
+
+```powershell-console
+PS C:\> Stop-Service wuauserv
+```
+
+To see the newly created events, type
+
+```powershell-console
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:155]: Combine these into one set of console output

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 12]
```diff
@@ -0,0 +1,209 @@
...
+PS C:\> $EventVariable=Get-Event
+```
+
+Look at the **MessageData**, it's the same message used in the Register-CimIndicationEvent
+
+```powershell
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:161]:
```powershell-console

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 13]
```diff
@@ -0,0 +1,209 @@
...
+PS C:\> $EventVariable
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:162]: PS> $EventVariable

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 14]
```diff
@@ -0,0 +1,209 @@
...
+
+To find the current state of this event
+
+```powershell-console
+PS C:\> $EventVariable.SourceEventArgs.NewEvent.SourceInstance
+```
+
+And the output looks like
+
+```powershell
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:184]: Remove these 4 linesd
also make the prompt 
PS>

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 15]
```diff
@@ -0,0 +1,209 @@
...
+PS C:\> $EventVariable.SourceEventArgs.NewEvent.SourceInstance
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:185]: PS> $EventVariable.SourceEventArgs.NewEvent.SourceInstance

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 16]
```diff
@@ -0,0 +1,209 @@
...
+0         wuauserv Manual    Stopped OK     0
+```
+
+To see the previous event state
+
+```powershell-console
+PS C:\> $EventVariable.SourceEventArgs.NewEvent.PreviousInstance
+```
+
+```powershell
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:198]: Remove these lines

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 17]
```diff
@@ -0,0 +1,209 @@
...
+PS C:\> $EventVariable.SourceEventArgs.NewEvent.PreviousInstance
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:199]: PS> $EventVariable.SourceEventArgs.NewEvent.PreviousInstance

**Review by doctordns — COMMENTED** [review]
I have made a lot of corrections and suggestgoins. but may have missed some.
1. the powershell code fencing is for pure powershell code, the powershell-console fencing  is to show output. IN some cases you have both, unneceessiarly.
2. I have changed to Pascal case in some cases
3. The prompt in blog articles is "PS>" - no drive letter.

I think I have it all - but take a look and lets look at the second draft.

Good work, by the way.

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md (hunk 3 above)
Comment (farismalaeb) [reply Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:88]: Is it ok to keep the select statement as I explain it, from my point of view it show the reader where the system class should placed and what is the number that comes after WITHIN...
or you prefer write it as **SELECT * FROM WITHIN WHERE TargetInstance ISA**
Thanks

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md (hunk 6 above)
Comment (farismalaeb) [reply Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:101]: Adding a link to it on Microsoft site, is it ok

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md (hunk 2 above)
Comment (farismalaeb) [reply Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md]: I added a link and also explain it

**Review by farismalaeb — COMMENTED** [review]
These are the new changes, I hope they are OK 
Thanks

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 18]
```diff
@@ -70,48 +79,44 @@ ProcessId Name                                     StartMode State   Status  Exi
...
-Select * from <WMI System Class> within <Number of Seconds> where TargetInstance ISA <WMI Class name>
-```
+>Select * from **WMI System Class** within **Number of Seconds** where TargetInstance ISA **WMI Class name**
 
-Let apply the same to *Win32_Serivce.* Start by Creating a PowerShell variable name it $Query and type the following query
+Let apply the same to *Win32_Serivce.* Start by creating a PowerShell variable, in our case, you construct the query as follows:
 
-```powershell-console
-PS C:\> $query = "Select * from CIM_InstModification within 10 where TargetInstance ISA 'Win32_Service'"
+```powershell
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:100]: coder fence should be powershell-console

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 19]
```diff
@@ -12,14 +13,15 @@ Q: How Can I Be Notified Any Time a Service Goes Down?
 
 A: The short quick answer to utilizing WMI and PowerShell 7.
 
-Using Powershell to create temporary event monitoring in the WMI classes,
-which keeps on monitoring any services changes and generates an alert once it detects a change.
+You use Powershell to create temporary event monitoring using WMI.
+Then WMI monitors any services changes and generates an alert once it detects a change.
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:17]: any service changes

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 20]
```diff
@@ -70,48 +79,44 @@ ProcessId Name                                     StartMode State   Status  Exi
...
+```powershell
+PS> $query = "Select * from CIM_InstModification within 10 where TargetInstance ISA 'Win32_Service'"
 ```
 
 > A full explanation for the WQL query is available in [Your Goto Guide for Working with Windows WMI Events and PowerShell](https://adamtheautomator.com/your-goto-guide-for-working-with-windows-wmi-events-and-powershell/)
 
 ## Registering The Query
 
-We have the WQL query, let move to the next step and register the query to the WMI events by using the **Register-CimIndicationEvent**
+We have the WQL query, let move to the next step and register the query to the WMI events by using the [**Register-CimIndicationEvent**](https://docs.microsoft.com/en-us/powershell/module/cimcmdlets/register-cimindicationevent?view=powershell-7.1).
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:108]: We have the WQL query, let's move to the next

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 21]
```diff
@@ -70,48 +79,44 @@ ProcessId Name                                     StartMode State   Status  Exi
...
+The **Register-CimIndicationEvent** is used to subscribe to events generated from the system.
+And in our case, it subscribes to events generated from the $query.
 
-```powershell-console
-PS C:\> Register-CimIndicationEvent -Namespace 'ROOT\CIMv2' -Query $query -SourceIdentifier 'WindowsServices' -MessageData 'Service Status Change'
+```powershell
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:112]: code fence should be powershell-console

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 22]
```diff
@@ -146,20 +147,16 @@ Running  wuauserv           Windows Update
...
-To see the newly created events, type
-
-```powershell-console
-PS C:\> $EventVariable=Get-Event
+```powershell
+PS> Stop-Service wuauserv
 ```
 
+To see the newly created events, type `Get-Event`
 Look at the **MessageData**, it's the same message used in the Register-CimIndicationEvent
```
Comment (doctordns) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:155]: Look at the **MessageData**, it's the same message used in the Register-CimIndicationEvent.

**Review by doctordns — CHANGES_REQUESTED** [review]
some very minor issues.

Also at the end of the article which Github won;t let me comment on"

This WMI monitoring remains active as long as the PowerShell console.
It creates such a temporary job which runs in the background to monitor the services class.
You can also end this process by rebooting the computer.
Hope you learned something new today.

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 23]
```diff
@@ -0,0 +1,186 @@
...
+
+You use Powershell to create temporary event monitoring using WMI.
+Then WMI monitors any service changes and generates an alert once it detects a change.
+
+## Basic Requirement
+
+To achieve this, you need Windows PowerShell 5.1 and above.
+
+This post uses the latest version of PowerShell 7.
+So if you are don't yet have PowerShell 7, see the [Microsoft documentation on how to install PowerShell 7 on Windows.](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows?view=powershell-7.1).
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:24]: When linking to docs, remove the locale (en-us) from the URL. Also remove them view parameter unless you are linking to a specific version of a cmdlet.

```suggestion
So if you are don't yet have PowerShell 7, see the Microsoft documentation on how to [Install PowerShell 7 on Windows](https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-windows).
```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 24]
```diff
@@ -0,0 +1,186 @@
+---
+post_title: How Can I Be Notified Any Time a Service Goes Down?
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:2]: Use sentence case

```suggestion
post_title: How can I be notified any time a service goes down?
```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 25]
```diff
@@ -0,0 +1,186 @@
...
+username: farisnt@gmail.com
+Categories: PowerShell
+tags: PowerShell, WMI, Events
+Summary: Using Powershell to create temporary event monitoring using WMI
+
+---
+
+**How Can I Be Notified Any Time a Service Goes Down?**
+
+Q: How Can I Be Notified Any Time a Service Goes Down?
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:12]: Remove redundant text and use sentence case.

```suggestion
Q: How can I be notified any time a service goes down?
```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 26]
```diff
@@ -0,0 +1,186 @@
...
+
+A: The short quick answer to utilizing WMI and PowerShell 7.
+
+You use Powershell to create temporary event monitoring using WMI.
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:16]:
```suggestion
You use PowerShell to create temporary event monitoring using WMI.
```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 27]
```diff
@@ -0,0 +1,186 @@
...
+
+Also, make sure that PowerShell is running as administrator.
+
+## Finding the Required Class
+
+Before going into details, you need to find the required class to monitor.
+To get a list of all available classes, use the following code.
+
+```powershell-console
+PS> Get-CimClass -Namespace root\cimv2
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:34]: Must use double backslash. This is a requirement imposed by the process that copies the post from GitHub to WordPress.

```suggestion
PS> Get-CimClass -Namespace root\\cimv2
```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 28]
```diff
@@ -0,0 +1,186 @@
...
+__InstanceProviderRegistration      {}                   {provider, InteractionType, QuerySupportLevels, SupportsBatch…
+__MethodProviderRegistration        {}                   {provider}
+__PropertyProviderRegistration      {}                   {provider, SupportsGet, SupportsPut}
+__EventConsumerProviderRegistration {}                   {provider, ConsumerClassNames}
+```
+
+The returned result represents all the available classes in the namespace.
+For this tutorial, the focus is on Windows services, which is represented by *Win32_Service*.
+
+To Enumerate the *Win32_Services* WMI class and get all the available services using PowerShell run the following code.
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:57]: Our style is to make class names bold.
```suggestion
For this tutorial, the focus is on Windows services, which is represented by **Win32_Service**.

To Enumerate the **Win32_Services** WMI class and get all the available services using PowerShell run the following code.
```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 29]
```diff
@@ -0,0 +1,186 @@
...
+
+```powershell-console
+PS> Get-CimInstance -Namespace root\CIMV2 -ClassName win32_service
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:60]:
```suggestion
PS> Get-CimInstance -Namespace root\\CIMV2 -ClassName win32_service
```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 30]
```diff
@@ -0,0 +1,186 @@
...
+- Create a WMI query language query.
+- Register this query.
+- Obtain any events generated.
+
+## Creating WMI Query
+
+WMI has many special classes that you can use to detect changes to other WMI classes.
+For example, you can use the **CIM_InstModification** class to monitor the targeted class, in this case **Win32_Service**
+
+You have to create a WMI query using [WMI Query Language](https://docs.microsoft.com/en-us/windows/win32/wmisdk/wql-sql-for-wmi).
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:93]:
```suggestion
You have to create a WMI query using [WMI Query Language](https://docs.microsoft.com/windows/win32/wmisdk/wql-sql-for-wmi).

```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 31]
```diff
@@ -0,0 +1,186 @@
...
+The WQL syntex structure looks like this:
+
+>Select * from **WMI System Class** within **Number of Seconds** where TargetInstance ISA **WMI Class name**
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:96]:
~~~suggestion
```
Select * from <WMI System Class> within <Number of Seconds> where TargetInstance ISA <WMI Class name>
```
~~~

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 32]
```diff
@@ -0,0 +1,186 @@
...
+
+Let apply the same to *Win32_Serivce.* Start by creating a PowerShell variable, in our case, you construct the query as follows:
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:98]:
```suggestion
Let apply the same to **Win32_Serivce**. Start by creating a PowerShell variable, in our case, you construct the query as follows:
```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 33]
```diff
@@ -0,0 +1,186 @@
...
+
+```powershell-console
+PS> $query = "Select * from CIM_InstModification within 10 where TargetInstance ISA 'Win32_Service'"
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:101]:
```suggestion
$query = "Select * from CIM_InstModification within 10 where TargetInstance ISA 'Win32_Service'"
```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 34]
```diff
@@ -0,0 +1,186 @@
...
+```
+
+> A full explanation for the WQL query is available in [Your Goto Guide for Working with Windows WMI Events and PowerShell](https://adamtheautomator.com/your-goto-guide-for-working-with-windows-wmi-events-and-powershell/)
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:104]:
```suggestion
A full explanation for the WQL query is available in [Your Goto Guide for Working with Windows WMI Events and PowerShell](https://adamtheautomator.com/your-goto-guide-for-working-with-windows-wmi-events-and-powershell/).
```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 35]
```diff
@@ -0,0 +1,186 @@
...
+
+## Registering The Query
+
+We have the WQL query, let's move to the next and register the query to the WMI events by using the [**Register-CimIndicationEvent**](https://docs.microsoft.com/en-us/powershell/module/cimcmdlets/register-cimindicationevent?view=powershell-7.1).
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:108]:
```suggestion
We have the WQL query, let's move to the next and register the query to the WMI events by using the [Register-CimIndicationEvent](https://docs.microsoft.com/powershell/module/cimcmdlets/register-cimindicationevent).
```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 36]
```diff
@@ -0,0 +1,186 @@
...
+The **Register-CimIndicationEvent** is used to subscribe to events generated from the system.
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:109]: Put cmdlet names in backticks.
```suggestion
The `Register-CimIndicationEvent` is used to subscribe to events generated from the system.
```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 37]
```diff
@@ -0,0 +1,186 @@
...
+And in our case, it subscribes to events generated from the $query.
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:110]: Put variables in backticks
```suggestion
And in our case, it subscribes to events generated from the `$query`.
```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 38]
```diff
@@ -0,0 +1,186 @@
...
+
+```powershell-console
+PS> Register-CimIndicationEvent -Namespace 'ROOT\CIMv2' -Query $query -SourceIdentifier 'WindowsServices' -MessageData 'Service Status Change'
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:113]:
```suggestion
Register-CimIndicationEvent -Namespace 'ROOT\\CIMv2' -Query $query -SourceIdentifier 'WindowsServices' -MessageData 'Service Status Change'
```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 39]
```diff
@@ -0,0 +1,186 @@
...
+```
+
+To confirm the successful registration, type the following cmdlet *Get-EventSubscriber,* the output looks like the following
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:116]: use backticks - punctuation goes outside the markup
```suggestion
To confirm the successful registration, type the following cmdlet `Get-EventSubscriber`, the output looks like the following
```

File: Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md [hunk 40]
```diff
@@ -0,0 +1,186 @@
...
+```
+
+So the service is running, let stop it by typing the following
+
+```powershell
+PS> Stop-Service wuauserv
+```
+
+To see the newly created events, type `Get-Event`
+Look at the MessageData, it's the same message used in the Register-CimIndicationEvent.
```
Comment (sdwheeler) [inline Posts/2021/07/How Can I Be Notified Any Time a Service Goes Down.md:155]: Bold object properties
```suggestion
Look at the **MessageData**, it's the same message used in the `Register-CimIndicationEvent`.
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]
See my suggestions. 
Take a look at the style guide and editorial checklist for our formatting rules:
- https://docs.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide
- https://docs.microsoft.com/powershell/scripting/community/contributing/editorial-checklist

**Comment by sdwheeler** [comment]
@doctordns @farismalaeb I just noticed that this article is in the July folder. Can you move it to the August folder?

**Comment by doctordns** [comment]
@farismaleb => can you move the file?

**Comment by farismalaeb** [comment]
Changes are committed and the post is moved to 08

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**